exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/CompoundIdentifierConverter.java

```diff
@@ -161,6 +171,7 @@ public SqlNode visitChild(
     rules.put(SqlJoin.class, R(D, D, D, D, D, E));
     rules.put(SqlOrderBy.class, R(D, E, D, D));
     rules.put(SqlDropTable.class, R(D));
+    rules.put(SqlSetOption.class, R(D, D, D));
```

Comment:
the comment doesn't explain why this specific rule has been added. It just explains how the rules work in general.
Why do we need this specific rule, what happens if I remove it ?